doc/api/fs.md

```diff
@@ -3314,7 +3318,10 @@ mkdtemp(tmpDir, (err, directory) => {
 // This method is *CORRECT*:
 import { sep } from 'node:path';
 mkdtemp(`${tmpDir}${sep}`, (err, directory) => {
-  if (err) throw err;
+   if (err) {
+    console.log(err)
+    return
+  }
```

Comment:
Why not throw here?

Reply:
If you were to go through the Node js docs Error class ,the section error first callbacks,talks about not throwing errors in error first callbacks,but I also noticed in the documentation it wasn't adhered to in other areas ,so I just changed a few to get a review from the maintainers ,then maybe it can be implemented all through.I think I wrote it in my commit message.

Reply:
OK I see where you come from, that [Error-first callback](https://nodejs.org/api/errors.html#error-first-callbacks) section of the docs would need some work as it's quite outdated and not very clear. The important piece of information is here: https://github.com/nodejs/node/blob/7df27582bbb896ebb72697da58209976a0b4daf6/doc/api/errors.md#L178-L180

Crashing the process is what I would want in this case (if the operation fails, I'd expect `node` to exit with a non-zero exit code); how to handle the error will depend on what your use case is, and I'm not convinced we can document it better.